coding-concepts.js

```diff
@@ -18,38 +18,38 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: my answer was 9
+// b) Verify and explain: .length reads the amount of characters from the string and gives you the answer
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: my answer is it will give is l 
+// b) Verify and explain: the index or [4] will count the position which is l
```

Comment:
Remember that index starts from 0, so when you look at the 4th index, it is actually the 5th character, which is "o"